src/Runner.Worker/action_yaml.json

```diff
@@ -131,7 +131,7 @@
                     "env": "step-env",
                     "working-directory": "string-steps-context",
                     "shell": {
-                        "type": "non-empty-string",
+                        "type": "string-shell-context",
```

Comment:
I think we should just do `string-steps-context` here. While most of those contexts may not be useful, there is also no reason to fail if someone finds a way to use `set-env` to set the `env` context to  the shell they want.

We will want to test all of these contexts in that list work as intended though, and we evaluate them properly

Reply:
Updated the context and tested. It seems like all of those contexts are available to us in shell now 😊